curriculum/challenges/english/16-the-odin-project/top-the-box-model/the-box-model-question-i.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 64a6749a7a7d00f97013ed10
+title: The Box Model Question I
+challengeType: 15
+dashedName: the-box-model-question-i
+--- 
+# --description--
+
+Because the box model concept is so incredibly fundamental, let’s dig a bit deeper with [this lesson from MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model#what_is_the_css_box_model). It covers the same material as the video above and will introduce you to inline boxes that we will explore in the next lesson. Pay close attention to the examples and take the time to experiment with their in-browser editor!
+
+# --question--
+
+## --text--
+
+What does the box-sizing CSS property do?
+
+## --answers--
+
+The standard box model calculates the width and height of an element based on the content alone, while the alternative box model calculates based on the content plus padding and border.
```

Comment:
What is `standard` vs `alternative`? I assume this verbiage is reflected in the missing(?) video, but we should probably directly explain the default `content-box` and the more commonly used `border-box`.